README.md

```diff
@@ -83,6 +83,16 @@ You can also insert a code snippet by typing the name of the snippet in your cod
 
 The COBOL Language Support extension also supports user snippets. Add your custom snippets to the `COBOL.json` file in your user snippets folder.
 
+### Syntax Analysis
```

Comment:
As we've decided to hide this setting from the UI part of settings, I wouldn't mention this option at all in docs. Please check with Aman if his change is part of the release